core/new-gui/src/index.html

```diff
@@ -9,10 +9,11 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
 
   <!-- Add Angular Material Fonts -->
-  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+  <!-- <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet"> -->
```

Comment:
what does this line do and why do we comment it out?

Reply:
Change it back. Accidentally commented it out when I was doing some configurations.